tests/test_fixture_exclusion.py

```diff
@@ -96,3 +96,26 @@ def test_invalid_exclude_patterns(names, values):
 ])
 def test_valid_exclude_patterns(names, values):
     slash.exclude(names, values)
+
+
+def test_exclude_on_fixture():
+
+    with slash.Session() as s:
+
+        @s.fixture_store.add_fixture
+        @slash.parametrize('param', [1, 2])
+        @slash.exclude("param", [1])
+        @slash.fixture
+        def fixture1(param):
+            return param
+
+        def test_something(fixture1):
```

Comment:
I think this test case should have an additional fixture that is parametrized and not marked with exclude to make sure this plays well together...

Reply:
Fixed